src/simparams.jl

```diff
@@ -140,7 +140,11 @@ function make_params(
   spreading_x0::Real=1,
   spreading_truncation::Real=Inf,
 
-  british_strain_multiplier::Real=1.70
+  british_strain_multiplier::Real=1.70,
+  delta_strain_multiplier::Real=1.7*1.5,
```

Comment:
why you need 2 places: here and `strains.jl` to specify multipliers?

Reply:
These are just default values. Perhaps we should just make them constant. What value do we use for the british variant and what for the delta variant?

Reply:
yes, constants you gave are correct